.eslintrc.js

```
@@ -41,7 +42,6 @@ module.exports = {
     'require-yield': 'error',
     'max-nested-callbacks': ['error', 4],
     'max-depth': ['error', 4],
-    'require-await': 'error',
```

Why did you delete this rule? with this everything works fine as you stated in this PR the goal is always return promise in case of `async` functions, and [this rule assures that](https://eslint.org/docs/rules/require-await) -> `The return value is always a Promise.`

---

I have not removed it but turned it off below. And no, everything does not work fine with this rule. This rule "Disallow async functions which have no await expression" which then leads to not using `async` and hence throwing exceptions instead of returning the Promise and to the whole reason that I am trying to fix here.

---

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/promise-function-async.md

This is the new rule that enforces the new behavior.

---

Hmm I originally thought, that this new rule required the turning off the `require-await` but I mixed that up with this rule: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/require-await.md

I am gonna test the Typescript's version of `require-await` to see if it would work correctly, but I don't think the original `require-await` works.